README.md

```diff
@@ -25,6 +25,11 @@ Running the site locally
 1. `docker-compose up --build -V`
 2. Open `http://localhost:8080` in your browser.
 
+Regenerating critical CSS
+-------------------------
+
+`docker-compose run critical_css`
```

Comment:
This is inconsistent with what the `Jenkinsfile` does, should it be updated to be one `run` and `one` exec at least?

Reply:
Also `Dockerfile.critical_css` seems to have no default `CMD` for this to use

Reply:
`run` can be used because of the volume (no `cp` needed).